lnbits/nodes/base.py

```diff
@@ -104,7 +104,7 @@ class NodePayment(BaseModel):
     fee: Optional[int] = None
     memo: Optional[str] = None
     time: int
-    bolt11: str
+    bolt11: Optional[str]
```

Comment:
shouldn't this add `= None` at the end like other optional fields above and below?

```suggestion
    bolt11: Optional[str] = None
```